package.json

```diff
@@ -32,17 +32,17 @@
         "@cparra/apexdocs": "^1.11.0",
         "@lwc/eslint-plugin-lwc": "^0.10.0",
         "@prettier/plugin-xml": "^0.12.0",
-        "@sa11y/jest": "^0.2.0",
+        "@sa11y/jest": "^0.2.1",
         "@salesforce/eslint-config-lwc": "^0.9.0",
-        "@salesforce/sfdx-lwc-jest": "^0.10.2",
+        "@salesforce/sfdx-lwc-jest": "^0.10.4",
         "babel-eslint": "^10.1.0",
-        "eslint": "^7.16.0",
+        "eslint": "^7.19.0",
         "eslint-config-prettier": "^7.2.0",
         "husky": "^4.3.8",
         "jest-canvas-mock": "^2.3.0",
```

Comment:
I just realized that you have the `jest-canvas-mock` lib in here and that's a Canvas API Jest mock which you don't need because you don't use canvas (that's my bad: I likely took this from LWC Recipes).
Can you drop it?